backend/scripts/load-test.js

```
@@ -1,7 +1,7 @@
-const autocannon = require("autocannon");
+const autocannon = require('autocannon');
```

ESLint

[error] Require statement not part of import statement.

(`@typescript-eslint/no-var-requires`)

---

[error] 'require' is not defined.

(no-undef)